src/differentElementArray.md

```diff
@@ -0,0 +1,52 @@
+# Different Element in Array
+This snip returns different elements between two arrays.
+
+## Result
+```
+ differentElemArray([1, 2, 3, 4, 5],[3, 4, 5, 6, 7]); //[1, 2]
+```
+
+## Snip Langauge
+* [JavaScript](#javascript)
+* [Python](#python)
+* [Java](#java)
+
+### JavaScript
+```js
+function arrayDiff(array1, array2) {
+    const diff = array1.filter((element) => !array2.includes(element)); 
+    return diff;
+}
+
+```
+### Python
+```python
+def differentElemArray(arr1, arr2):
+  res = list(set(arr1) - set(arr2))
+  return res
+```
+
+### Java
+```java
+    import java.util.*;
+
+    class JavaExample{
+        
+    
+        public static void diffElem(List list1,List list2){
```

Comment:
Update the function name to match it with the example result

Reply:
The name is updated